xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -30,8 +31,8 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Initializes a new instance of the <see cref="TextEncoder"/> class.</summary>
```

Comment:
these cref links in the summary need to have the full asset it, such as, T:System.Text.Encodings.Web.TextEncoder or it doesn't render as a link